test/Interpreter/generic_subscript_static.swift

```diff
@@ -0,0 +1,142 @@
+//===--- generic_subscript.swift ------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
```

Comment:
Nitpick: 2019

Reply:
Updated to 2014 - 2019, since these tests are basically modified versions of another test file dating back that far.